transform/mattermost-analytics/models/marts/product/trial_requests/fct_trial_request_history.sql

```diff
@@ -40,6 +40,9 @@ select
     , tr.end_at
     , tr.request_source
     , tr.request_type
+    , tr.product_id
```

Comment:
Shall we create a `dim_products`? I'm a bit uncertain as we'll need to figure out sources outside Stripe and correlate them. This will mean that a separate ID will need to be added.

Reply:
I think it make sense eventually - I see Stripe, CWS (though still calles stripe_product_id) and SFDC as current sources. For now we can use this one and revisit once we hit in more places.